warhorn/warhorn.py

```diff
@@ -1103,6 +1070,21 @@ def get_base_path(node_name="warriorframework", **kwargs):
     return base_path, node
 
 
+def activate_virtualenv(node, destination, logfile, print_log_name):
+    '''Activate virtual environment to add dependencies
+    '''
+    ve_name = get_attribute_value(node, 'name')
+    ve_loc = get_attribute_value(node, 'location')
+    ve_dest = destination + ve_name
+    print_info("ve_name: "+ve_name, logfile, print_log_name)
+    print_info("destination: "+ve_dest, logfile, print_log_name)
+    venv_cmd = os.path.expanduser(ve_loc)
+    subprocess.call([venv_cmd, "--system-site-packages", ve_dest])
+    venv_file = "{}/bin/activate_this.py".format(ve_dest)
+    execfile(venv_file, dict(__file__=venv_file))
+    return True
```

Comment:
This function shouldn't only return True, there are multiple operations in here that can go wrong